applications/GeoMechanicsApplication/custom_constitutive/mohr_coulomb_with_tension_cutoff.hpp

```diff
@@ -0,0 +1,91 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//
+//  Main authors:    Mohamed Nabi,
+//                   Wijtze Pieter Kikstra
+// //
+
+#pragma once
+
+// System includes
+#include <cmath>
+
+// Project includes
+#include "custom_constitutive/coulomb_yield_surface.hpp"
+#include "custom_constitutive/tension_cutoff.hpp"
+#include "geo_mechanics_application_variables.h"
+#include "includes/constitutive_law.h"
+#include "includes/serializer.h"
+
+namespace Kratos
+{
+class ConstitutiveLawDimension;
```

Comment:
Add a forward declaration here:
```suggestion
class ConstitutiveLawDimension;
class Serializer;
```